libsupport/include/katana/ProgressTracer.h

```diff
@@ -81,6 +81,17 @@ class Value : public variant_type {
 };
 using Tags = std::vector<std::pair<std::string, Value>>;
 
+struct HostStats {
+  long nprocs{};
+  long ram_gb{};
+  std::string hostname;
+};
+
+uint64_t ParseProcSelfRssBytes();
+HostStats GetHostStats();
+long GetMaxMem();
+std::string GetValue(const Value& value);
```

Comment:
Could these be static members of `ProgressTracer` instead? (And if not we definitely need a more descriptive name for this global function than `GetValue`)